src/providers/CourtConfig.js

```diff
@@ -0,0 +1,41 @@
+import React, { useContext } from 'react'
+import PropTypes from 'prop-types'
+import { useSubscription } from 'urql'
+import { CourtConfig as CourtConfigSubscription } from '../queries/courtConfig'
+
+import environment from '../environment'
+import { getNetworkName } from '../web3-utils'
+import { networks } from '../networks'
+
+const CHAIN_ID = environment('CHAIN_ID')
+const CourtConfigContext = React.createContext()
+
+function CourtConfigProvider({ children }) {
+  const courtAddress = networks[getNetworkName(CHAIN_ID)].court
+
+  console.log(courtAddress)
+
+  const [result] = useSubscription({
```

Comment:
i am using one hook per query so in the component we just use `data = useCourtSubscription`